src/components/balance/ModalTransferAmount.vue

```diff
@@ -184,31 +201,195 @@
+import { MutationTypes } from '@/store/mutation-types';
+import { ActionTypes } from '@/store/action-types';
+import { getUnitNames, getUnit } from '@/helper/units';
 import ModalSelectAccountOption from '@/components/balance/ModalSelectAccountOption.vue';
+import FormatBalance from '@/components/balance/FormatBalance.vue';
 import IconBase from '@/components/icons/IconBase.vue';
-import IconSolidChevronDown from '@/components/icons/IconSolidChevronDown.vue';
+// import IconSolidChevronDown from '@/components/icons/IconSolidChevronDown.vue';
 import IconAccountSample from '@/components/icons/IconAccountSample.vue';
 import IconSolidSelector from '@/components/icons/IconSolidSelector.vue';
-import IconExchange from '@/components/icons/IconExchange.vue';
+// import IconExchange from '@/components/icons/IconExchange.vue';
 
 export default defineComponent({
   components: {
     ModalSelectAccountOption,
     IconBase,
-    IconSolidChevronDown,
+    // IconSolidChevronDown,
     IconAccountSample,
     IconSolidSelector,
-    IconExchange,
+    // IconExchange,
+    FormatBalance,
+  },
+  props: {
+    allAccounts: {
+      type: Array,
+      required: true,
+    },
+    allAccountNames: {
+      type: Array,
+      required: true,
+    },
+    address: {
+      type: String,
+      required: true,
+    },
+    accountIdx: {
+      type: Number,
+      required: true,
+    },
+    balance: {
+      type: BN,
+      required: true,
+    },
   },
   setup(props, { emit }) {
     const closeModal = () => {
       emit('update:is-open', false);
     };
 
+    const openOption = ref(false);
+
+    const unitToken = inject('unitToken', '');
+    const decimal = inject('decimal', 10);
+
+    const selAccount = ref(0);
+    const transferAmt = ref(0);
+
+    const toAccount = ref(props.allAccounts[0] as string);
+    const toAddress = ref(props.allAccounts[0] as string);
+    const toAccountName = ref(props.allAccountNames[0]);
+
+    const arrUnitNames = getUnitNames(unitToken);
+    const selectUnit = ref(unitToken);
+
+    // const { balance } = useBalance(defaultAccount);
+    // provide('balance', balance);
+
+    const formatBalance = computed(() => {
+      const tokenDecimal = decimal;
+      return plasmUtils.reduceBalanceToDenom(
+        props.balance.clone(),
+        tokenDecimal
+      );
+    });
+
+    const { api } = useApi();
+    const store = useStore();
+
+    const transfer = async (transferAmt: number, selAddress: string) => {
+      console.log('transfer', transferAmt);
+      console.log('selAccount', selAddress);
+      console.log('selUnit', selectUnit.value);
+
+      if (Number(transferAmt) === 0) {
+        store.dispatch(
+          ActionTypes.SHOW_ALERT_MSG,
+          `The amount of token to be transmitted must not be zero`
+        );
+        return;
+      } else if (!plasmUtils.isValidAddressPolkadotAddress(selAddress)) {
+        store.dispatch(ActionTypes.SHOW_ALERT_MSG, `The address is not valid`);
```

Comment:
The alert toast is green (indicating success). Can you add the ability to choose different alert types? (like error, warning, success)